app/views/admin/submissions/show.html.erb

```diff
@@ -23,6 +23,7 @@
           </div>
           <%= render 'admin/consignments/consignment_section', consignment: @submission.consigned_partner_submission, artist: @submission.artist&.name %>
           <%= render 'admin/offers/offers_section', offers: @offers, truncated: true %>
+          <%= render 'admin/submissions/notes_section', notes: @notes, truncated: true %>
```

Comment:
This could probably be more straight forward for Rails users other than adding our own pattern here:

```erb
<% if @notes.size > 0 %>
  <div class='notes-section'>
    <div class='notes-section-title'>
      Notes
    </div>

    <%= render @notes %>
  </div>
<% end %>
```

`app/views/admin/notes/_note.html.erb`:

```erb
<div class='list-group-item--note'>
  <div class='list-group-item--byline'>
    <%= note.byline %>
  </div>
  <div class='list-group-item--body'>
    <textarea>
      <%= note.body %>
    </textarea>
  </div>
</div>
```

And remove related partials.

Reply:
Awesome, did not know this was possible